src/main/java/org/threadly/concurrent/SingleThreadScheduler.java

```diff
@@ -246,29 +247,40 @@ protected void finalize() {
    * @author jent - Mike Jensen
    * @since 2.0.0
    */
-  protected static class SchedulerManager extends AbstractService 
-                                          implements Runnable {
+  protected static class SchedulerManager implements Runnable {
```

Comment:
You can see we no longer extend AbstractService and "shutdownFinished" variable is also gone as it is all represented in the AtomicInteger "state"